config/scanner/scanner_test.go

```diff
@@ -72,40 +72,40 @@ func TestScanner_ScanConfig(t *testing.T) {
 				FilePath: "main.tf",
 				Successes: []types.MisconfResult{
 					{
-						Message: "Resource 'aws_security_group_rule.my-rule' passed check: An egress security group rule allows traffic to /0.",
+						Message: "Resource 'aws_security_group_rule.my-rule' passed check: Ensures that usage of security groups with inline rules and security group rule resources are not mixed.",
 						PolicyMetadata: types.PolicyMetadata{
 							Type:               "Terraform Security Check powered by tfsec",
-							ID:                 "AWS007",
-							Description:        "Your port is egressing data to the internet",
-							RecommendedActions: "Set a more restrictive cidr range",
-							Severity:           "CRITICAL",
+							ID:                 "AVD-AWS-0100",
+							Description:        "Security group rules will be overwritten and will result in unintended blocking of network traffic",
+							RecommendedActions: "Either define all of a security group's rules inline, or none of the security group's rules inline",
+							Severity:           "LOW",
 						},
 					},
 					{
-						Message: `Resource 'variable.enableEncryption' passed check: Potentially sensitive data stored in "default" value of variable.`,
+						Message: `Resource 'aws_security_group_rule.my-rule' passed check: An egress security group rule allows traffic to /0.`,
 						PolicyMetadata: types.PolicyMetadata{
 							Type:               "Terraform Security Check powered by tfsec",
-							ID:                 "GEN001",
-							Description:        "Default values could be exposing sensitive data",
-							RecommendedActions: "Don't include sensitive data in variable defaults",
+							ID:                 "AVD-AWS-0104",
+							Description:        "Your port is egressing data to the internet",
+							RecommendedActions: "Set a more restrictive cidr range",
 							Severity:           "CRITICAL",
 						},
 					},
 					{
-						Message: `Resource 'aws_security_group_rule.my-rule' passed check: Potentially sensitive data stored in block attribute.`,
+						Message: "Resource 'aws_security_group_rule.my-rule' passed check: Potentially sensitive data stored in block attribute.",
 						PolicyMetadata: types.PolicyMetadata{
+							ID:                 "AVD-GEN-0001",
 							Type:               "Terraform Security Check powered by tfsec",
-							ID:                 "GEN003",
 							Description:        "Block attribute could be leaking secrets",
-							RecommendedActions: "Don't include sensitive data in blocks",
 							Severity:           "CRITICAL",
+							RecommendedActions: "Don't include sensitive data in blocks",
 						},
 					},
 					{
 						Message: `Resource 'azurerm_managed_disk.source' passed check: Potentially sensitive data stored in block attribute.`,
 						PolicyMetadata: types.PolicyMetadata{
 							Type:               "Terraform Security Check powered by tfsec",
-							ID:                 "GEN003",
+							ID:                 "AVD-GEN-0001",
```

Comment:
I couldn't find this ID in AVD. Is it planned to be added?
https://avd.aquasec.com/

Reply:
It's here - https://github.com/aquasecurity/cloud-metadata/blob/main/general/secrets/sensitive-in-attribute/metadata.json 

The idea of this repo is that all IAC and CSPM rules will share the same set of common metadata but have different remediation steps depending on the reason (cli to resolve issues already in production maybe, or the required changes to terraform or cloudformation to fix the issue in code)

Reply:
Do you mean AVD will not have the ID? Should Trivy link these IDs to TFsec doc directly? I thought all information would be aggregated in AVD and Trivy should link IDs to AVD.

Reply:
OK, thanks!

Reply:
Trivy currently refers to the tfsec documentation (tfsec.dev). In other words, this change would not be a regression in terms of references. I think we can merge this PR now and will switch references to AVD once AVD will be ready.